src/main/java/nl/novi/TechItEasy/models/Television.java

```diff
@@ -30,10 +33,17 @@ public class Television {
     private Integer originalStock;
     private Integer sold;
 
+    @OneToOne
+    @JoinColumn(name="remote_controller_id")
+    private RemoteController remoteController;
+
     @ManyToOne
```

Comment:
Je kan hierachter nog (fetch = FetchType.eager) achter zetten zodat bij het ophalen van de gegevens de Cimodule direct meekomt.

Reply:
Ik zou FetchType.Eager niet aanraden als standaard. Gebruik dat alleen als je twee entiteiten hebt die onlosmakelijk met elkaar verbonden zijn en het belangrijk is dat je die informatie er altijd bij hebt, zoals een User en een Profile of een Book en een Author.